<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>965</width>
    <height>516</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QtHomework - Classes</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(85, 87, 83); border-radius: 0</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="classMenuButton">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: rgb(211, 215, 207)
}
QPushButton:hover {
	background-color: rgb(211, 215, 207)
}</string>
    </property>
    <property name="text">
     <string>Classes</string>
    </property>
   </widget>
   <widget class="QPushButton" name="homeworkMenuButton">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>30</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: rgb(186, 189, 182)
}
QPushButton:hover {
	background-color: rgb(211, 215, 207)
}</string>
    </property>
    <property name="text">
     <string>Homework</string>
    </property>
   </widget>
   <widget class="QPushButton" name="calendarMenuButton">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>60</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: rgb(186, 189, 182)
}
QPushButton:hover {
	background-color: rgb(211, 215, 207)
}</string>
    </property>
    <property name="text">
     <string>Calendar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="settingsMenuButton">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>90</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: rgb(186, 189, 182)
}
QPushButton:hover {
	background-color: rgb(211, 215, 207)
}</string>
    </property>
    <property name="text">
     <string>Settings</string>
    </property>
   </widget>
   <widget class="QLabel" name="backgroundLeftMenu">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>121</width>
      <height>501</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
	background-color: rgb(186, 189, 182)
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <zorder>backgroundLeftMenu</zorder>
   <zorder>classMenuButton</zorder>
   <zorder>homeworkMenuButton</zorder>
   <zorder>calendarMenuButton</zorder>
   <zorder>settingsMenuButton</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>965</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
